src/substrait/common/CMakeLists.txt

```diff
@@ -10,6 +10,15 @@ target_link_libraries(
         substrait_common
         fmt::fmt-header-only)
 
+add_library(
```

Comment:
we already have the substrait_type library under the substrait/type directory, like I mentioned previous, can we place the Decimal under substrait/expression directory ?

Reply:
@chaojun-zhang I definitely can see the argument for not placing it in the common directory and I missed that we had a similarly named library already so I'm ready to move them.  The substrait/expression directory does not yet exist -- would you like me to move this file to start it?

Reply:
Done.